### PR TITLE
Layer wise learning rate multipliers

### PR DESCRIPTION
**Pull request by AvantiShri**
Syntax is that in the layer constructor, you just add "W_learning_rate_multiplier=..." or "b_learning_rate_multiplier=...".

I ran tests/test_sequential_model.py, tests/test_optimizers.py and tests/keras/layers/*; I didn't run a comprehensive set of tests because some of them (like test_graph_model.py) appeared to be failing on both the master branch and my branch. However, the tests I ran should cover everywhere I changed code, and I also ran my own independent (although not automated) tests (and ran even more when I first created this branch)

### CODE REVIEW

**Comment by AvantiShri** [comment]
Sorry, I was trying to merge into my lab's keras fork and accidentally created a pull request to the real keras repo.
